.travis.yml

```diff
@@ -13,14 +12,19 @@ env:
   - GMOCK_PATH=/usr/src/gmock #1.6.0 from ubuntu trusty repo
 matrix:
   exclude:
-    - os: osx
-      env: GMOCK_PATH=/usr/src/gmock
-    - os: osx
-      compiler: gcc #does anyone on osx use it?
+    - os: linux
+      compiler: gcc
+      env: GMOCK_VER=1.8.0
```

Comment:
I thought it wasn't necessary as gmock 1.8 is used for the valgrind build. Reverted it now.